packages/Stylers/stories/helpers.scss

```diff
@@ -70,7 +70,7 @@
       @for $i from 0 through 7 {
         &:nth-child(#{$i + 1}) {
           left: $offset * $i;
-          top: $offset / 4 * $i;
+          top: calc(#{$offset / 4 * $i});
```

Comment:
`calc()` means value will be evaluated in runtime. We can it in compilation time. We just need to express division through multiplication: `$offset * 0.25 * $i`